README.md

```diff
@@ -170,3 +170,6 @@ To triage it, trigger LSP renaming using `:lua vim.lsp.buf.rename()` while the c
 ## Development
 
 Useful commands are defined in the [`Justfile`](Justfile) and can be listed with [`just`](https://github.com/casey/just).
+
+### Required packages to run tests:
+- npm install -g typescript-language-server typescript
```

Comment:
```suggestion
### Required packages to run tests

```console
npm install -g typescript-language-server typescript
```console
```

Reply:
This is easier to grasp in the README and easier to copy/paste.

Reply:
Going to fix the git history for this PR. I tried another merge strategy and seems the result was not as expected. Will try another one. Sorry, although I had use git for many years, this is my first OS collaboration and I'm learning about keeping clean the git history

Thanks for noticing the bash error 

This PR is near to be finished. It already works locally and I'm fixing the pipeline